src/JustSaying/JustSayingBus.cs

```diff
@@ -138,23 +135,23 @@ public void AddQueue(string region, string consumerGroup, ISqsQueue queue)
             publishersByType[topicType] = messagePublisher;
         }
 
-        private Task _consumerCompletionTask;
-        private bool _consumerStarted;
+        private Task _subscriberCompletionTask;
+        private bool _subscriberStarted;
 
         public Task Start(CancellationToken stoppingToken)
         {
             if (stoppingToken.IsCancellationRequested) return Task.CompletedTask;
 
             // Double check lock to ensure single-start
-            if (_consumerStarted) return _consumerCompletionTask;
+            if (_subscriberStarted) return _subscriberCompletionTask;
```

Comment:
This block could do with the same refactor as the other code that reduced the number of duplicated `return` statements.